### DISCUSSION TITLE
15th Feb to 29th Feb Integrations Report - Furqan Ahmed

### DISCUSSION THREAD

**Discussion by nblogist** [ringdao-community-guild/discussion#101] [General]
Hello community!

Here's a summary of the major work done on the integration front in the period mentioned above

**- Integration with Revoke.Cash**

Revoke.cash is a tool that allows users to manage and revoke token approvals in their cryptocurrency wallets, enhancing security by preventing unauthorized access to their digital assets. It was proposed to integrate Darwinia and Carb network making it easy for our community to revoke allowance to DApps present on our networks.
This was completed and a PR is currently open and we are waiting for new crab.subview.xyz RPC node to sync after that the PR will be merged: 
[PR](https://github.com/RevokeCash/revoke.cash/pull/195)
[Discussion](https://github.com/orgs/dcdao/discussions/86)

**- Update network Information on 3rd Party Platforms**
It is important to keep the details about the project relevant and updated accross the board - keeping it consistent for users, community and most importantly PR/media people. I had help from @1022ranji on this task and wanted to give kudos to her for the immaculately organized data she provided.
We complied and unified all the narrative that we need to be on the internet and you can find the whole discussion on the discussion link below, three websites were found to be priority in updation: CoinGecko, CMC, Parachain.info ; Parachain.info was requested and updated, you can find the details [here](https://parachains.info/details/darwinia#about).
[Unified Narritive](https://github.com/orgs/dcdao/discussions/92)
[Discussion](https://github.com/orgs/dcdao/discussions/78)
**Help Needed**: I wasn't able to update most of these sites cause I didn't have proof of association with Darwinia.

**- Support for Darwinia on Openzeppelin Defender**
Thanks to @hackfisher for introducing with Openzepplin folks, But they don't seem to be responding on the group after several pings as well, @hackfisher might have to press them about it. I have submitted their form already [here](https://github.com/orgs/dcdao/discussions/85#discussioncomment-8492728).
[Discussion](https://github.com/orgs/dcdao/discussions/85)

Feel free to add any questions and comments you might have, happy to hear any feedback too!

CC @hackfisher @wuminzhe

**Comment by nblogist** [comment]
Update: Revoke.cash finally has Darwinia and Crab Network Integrated! https://github.com/RevokeCash/revoke.cash/pull/195

**Comment by hackfisher** [comment]
Thank you for the update. It's great to see our integration efforts progressing.

In response to our evolving needs, I've suggested some role changes to the DCDAO Steering Committee:

1. Transition @casinoyoda to DCDAO Treasury Support, allowing for additional focus on data and AI if possible. @casinoyoda is currently member of the DCDAO Steering Committee, he welcomes this adjustment. The new title will better reflect his current scope of work.

2. With @casinoyoda's role change, the position of Technical Manager will become vacant. I believe Furqan is well-suited for this role, especially to oversee the Ecosystem Program, leveraging his integration experience, technical expertise, and leadership skills. Furqan's ability to commit more time and his network of resources make him ideal for managing both the technical and management aspects, including recruiting data analysts to advance the Ecosystem Program, and assisting DCDAO in contributing to the Darwinia website.

These changes aim to align our team structure with current demands, enhancing our efficiency and effectiveness. @wuminzhe will discuss the details with @nblogist  next week to advance this suggestion. Have a great weekend!

**Reply by nblogist** [reply]
Thank you @hackfisher for the review, I really appreciate it; Looking forward to contributing more! :rocket: 
Congrats on the new role @casinoyoda ! :partying_face:
